fix(k8s-infra-operator): support for 1.22

paologallinaharbur opened the pull request:
#### Is this a new chart
no
#### What this PR does / why we need it:
Fix https://github.com/newrelic/helm-charts/issues/721, related to https://github.com/jet/kube-webhook-certgen/issues/30

#### Which issue this PR fixes
[*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #](https://github.com/newrelic/helm-charts/issues/721)

#### Special notes for your reviewer:

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

roobre commented on charts/newrelic-infra-operator/values.yaml:
```diff
@@ -23,8 +23,8 @@ image:
-    repository: jettech/kube-webhook-certgen
-    tag: v1.5.0
+    repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
+    tag: v1.1.1
```
Small discrepancy, above we say `v1.1.0`.

Related: Did the ingress-nginx fellas _downgrade_ the version? Seems strange but might be okay.

paologallinaharbur replied:
it is v1.1.1 somehow I messed up copying pasting 😖 
https://github.com/kubernetes/k8s.io/blob/055cf8523bdbf31c00a47d20f057fc2c231edbb7/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml#L126

paologallinaharbur replied:
Fixed it and I even remembered adding the Co-authored 🤯

kang-makes replied:
Are we deprecating any Kubernetes version with this change that we must be aware of?

paologallinaharbur replied:
No, this image is already be used by different charts. All changes are related to the already in place <1.15 deprecation

kang-makes reviewed (approved):
LGTM.

I wonder why are we changing from jettech's webhook to the ingress-nginx but it seems cool.

paologallinaharbur commented:
Basically we moved because of this https://groups.google.com/g/kubernetes-sig-network/c/gRIKiI1JSTo?pli=1 (that change ended up being accepted)
the old repo got discontinued and incorporated inside ingress-nginx
